### ISSUE TITLE
icetray-inspect (Trac #1928)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1920]
icetray-inspect is definitely not returning all services and projects available.  

In some instances (icetray) I see c++ modules, and in
others (phys-services) you only see things from python listed.  
No information from services is listed (why I was looking originally)

Cweaver speculated that this was a known bug that only returned python results if they were there.  

This is icerec/trunk (so icetray/trunk)

Migrated from https://code.icecube.wisc.edu/ticket/1928
```json
{
    "status": "closed", 
    "changetime": "2019-01-12T00:29:07", 
    "description": "icetray-inspect is definitely not returning all services and projects available.  \n\nIn some instances (icetray) I see c++ modules, and in\nothers (phys-services) you only see things from python listed.  \nNo information from services is listed (why I was looking originally)\n\nCweaver speculated that this was a known bug that only returned python results if they were there.  \n\nThis is icerec/trunk (so icetray/trunk)\n\n", 
    "reporter": "blaufuss", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547252947290711", 
    "component": "combo core", 
    "summary": "icetray-inspect", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-12-21T20:45:37", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20161225T15:51:18: 
* kjmeagher changed status from "new" to "closed"
* kjmeagher changed resolution from "" to "fixed"
* kjmeagher commented:

> In changeset 2766:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2766"
> fix icetray inspect to print cpp modules for projects with dashes in their name, fixes #1920
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190111T20:52:31: nwhitehorn commented:

> In changeset 168783:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168783"
> fix icetray inspect to print cpp modules for projects with dashes in their name, fixes #1920
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190112T00:29:07: nwhitehorn commented:

> In changeset 170362:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170362"
> fix icetray inspect to print cpp modules for projects with dashes in their name, fixes #1920
> ```
